addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -322,7 +325,7 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
                     "serverInfo": self._get_rtc_server_info(rtc_session, ice_servers),
                 },
             )
-        if len(self.channel_id.rtc_session_ids) == 1 and self.channel_id.channel_type != "channel":
+        if len(self.channel_id.rtc_session_ids) == 1 and self.channel_id.channel_type != "channel" and self._should_invite_members():
```

Comment:
I would have moved the condition inside the new method, so people are free to fully override it or override it with "super and" as they see fit.